gst/rtsp-server/rtsp-media.h

```diff
@@ -451,6 +458,12 @@ void                  gst_rtsp_media_set_rate_control (GstRTSPMedia * media, gbo
 GST_RTSP_SERVER_API
 gboolean              gst_rtsp_media_get_rate_control (GstRTSPMedia * media);
 
+GST_RTSP_SERVER_API
+GstElement *          gst_rtsp_media_get_pipeline (GstRTSPMedia * media);
```

Comment:
is this relevant to this commit? this string doesn't appear elsewhere in the diff, see also get_bin below

Reply:
Yes agreed, this should be removed